src/App.js

```diff
@@ -9,22 +9,38 @@ import './assets/styles.css';
 export default function App() {
 
   return (
-      <div>
+      <div className='main'>
+        
         
           <Navbar />
-            <br></br>
-            <br></br>
-            <br></br>
-            <br></br>
-            <br></br>
-            <div>
-                <h1>WELCOME</h1>
-               
+          
+           <div className='hero'>
+          
+               <div className='frame'> 
+                 <br></br>
+                 <br></br>
+
+                <h1 className='title'>Όλοι οι αγώνες σε ένα σημείο!</h1>
+                <h4 className='subtitle'>Η #1 πλατφόρμα αναζήτησης αγώνων δρόμου στην Ελλάδα!</h4>
+
+                <div className='ActionBtns'>
+                  <button className='CTAbtn' > Δες τους αγώνες </button>
```

Comment:
```suggestion
                  <button className='CTAbtn'>Δες τους αγώνες</button>
```
Δε χρειάζονται κενά. Έτσι θα είναι ομοιόμορφο με τον υπόλοιπο κώδικα και επίσης τα κενά στο κείμενο δεν έχουν καμία επίπτωση στην τελική εμφάνιση.